src/components/NumberTicker.tsx

```diff
@@ -2,36 +2,33 @@ import * as React from 'react'
 import { Animated, StyleProp, StyleSheet, Text, TextStyle, View } from 'react-native'
 import { typeScale } from 'src/styles/fonts'
 
-interface CommonProps {
-  textHeight: number
-  textStyles?: StyleProp<TextStyle>
-  animationDuration?: number
-}
-interface Props extends CommonProps {
+interface Props {
   finalValue: string
+  typeScaleName?: keyof typeof typeScale
+  animationDuration?: number
   testID?: string
 }
 
-interface TickProps extends CommonProps {
-  startValue: number
-  endValue: number
+interface TickTextProps {
+  textStyles: StyleProp<TextStyle>
+  value: string
 }
 
-interface TickTextProps extends CommonProps {
-  value: string
+interface TickProps {
+  startValue: number
+  endValue: number
+  animationDuration: number
+  textHeight: number
+  textStyles: StyleProp<TextStyle>
```

Comment:
super nit: i wonder why it's plural?

Reply:
that's true, i'm not sure why i got into the habit of using plural for styles when they are props. changed it to singular